services/QuillLMS/client/app/bundles/Connect/styles/components/_student-container.scss

```diff
@@ -26,3 +26,7 @@
   width: 830px;
   box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
 }
+
+.student-container hr {
+  background-color: $hr-gray;
```

Comment:
This is actually defined in our outermost variables.scss file as `$quill-grey-10`, so you can just use that instead of having to define it independently in the Connect variable scss file.